### ISSUE TITLE
`box` command not found

### ISSUE THREAD

**Issue by benji07** [manala/manalize#91]
After a clean installation with php 7.1, Vbox 5.1.10 & Vagrant 1.9.0

```
➜  manalize git:(master) make build
Loading composer repositories with package information
Updating dependencies
Dependency resolution completed in 0.129 seconds
Analyzed 3130 packages to resolve dependencies
Analyzed 50606 rules to resolve dependencies
Dependency resolution completed in 0.000 seconds
  - Installing elao/enum (v0.4.0)
    Downloading
    Extracting archive

  - Installing composer/semver (1.4.2)
    Downloading
    Extracting archive

  - Installing nikic/iter (v1.4.0)
    Downloading
    Extracting archive

  - Installing guzzlehttp/promises (1.3.0)
    Downloading
    Extracting archive

  - Installing psr/http-message (1.0.1)
    Downloading
    Extracting archive

  - Installing guzzlehttp/psr7 (1.3.1)
    Downloading
    Extracting archive

  - Installing guzzlehttp/guzzle (6.2.2)
    Downloading
    Extracting archive

  - Installing symfony/yaml (v3.2.0)
    Downloading
    Extracting archive

  - Installing symfony/process (v3.2.0)
    Downloading
    Extracting archive

  - Installing symfony/filesystem (v3.2.0)
    Downloading
    Extracting archive

  - Installing psr/log (1.0.2)
    Downloading
    Extracting archive

  - Installing symfony/debug (v3.2.0)
    Downloading
    Extracting archive

  - Installing symfony/polyfill-mbstring (v1.3.0)
    Downloading
    Extracting archive

  - Installing symfony/console (v3.2.0)
    Downloading
    Extracting archive

Writing lock file
Generating optimized autoload files
/bin/sh: box: command not found
make: *** [build] Error 127
```

**Comment by chalasr** [comment]
Yeah, right now we can't add `box` as a dev dependency because of https://github.com/box-project/amend/issues/12. What you need to fix this is to install https://github.com/box-project/box2#as-a-phar-recommended globally on your host.

I'll retry to require it to see what happens. Anyway, let's use this as a reminder by keeping open.
Thanks @benji07

**Comment by chalasr** [comment]
Btw if you need to install `manalize` for your personal use you should consider using the [installer](https://github.com/manala/manalize#using-the-installer-recommended)

**Comment by chalasr** [comment]
Should be fixed by #92
